packages/iTwinUI-react/src/core/NonIdealState/ErrorPage.tsx

```diff
@@ -6,17 +6,15 @@ import React from 'react';
 import { Button } from '../Buttons/Button';
 import { NonIdealState } from './NonIdealState';
 import { CommonProps } from '../utils';
-import {
-  Svg401,
-  Svg403,
-  Svg404,
-  Svg500,
-  Svg502,
-  Svg503,
-  SvgError,
-  SvgRedirect,
-  SvgTimedOut,
-} from '@itwin/itwinui-illustrations-react';
+import Svg401 from '@itwin/itwinui-illustrations-react/cjs/illustrations/401';
```

Comment:
what's the reason for this change?

there is really _no_ need to import from internal cjs paths anymore, whether it's illustrations in this component or icons in stories/playgrounds.

Reply:
For the icons, I was making these changes to match the [STYLEGUIDE.md](https://github.com/iTwin/iTwinUI-react/blob/main/STYLEGUIDE.md#import-individual-icons-directly-from-the-cjs-module) recommendation.

![image](https://user-images.githubusercontent.com/45748283/207429008-3c57489d-8b09-4475-9fd1-ac0fdeb081ea.png)

For the illustrations, there was no style guide recommendation. So I thought of adding a recommendation to import illustrations similar to icons. I mentioned this in the PR desc.

But if there is no benefit for importing icons and illustrations from cjs, then I can undo these changes and remove those recommendations from the style guide. So, contributors can import icons and illustrations in whatever way they prefer.

Reply:
yes, the task is to update the styleguide which has become outdated after #912. for more context, you can see the history in #917 and #924.

Reply:
Looks like i didn't clearly document the thought process in those PRs so i'll explain:

In v1, we were relying on `@itwin/itwinui-icons-react`. This is a huge package with tons of icons, so treeshaking is very important to reduce client bundle size. But treeshaking was not working correctly with module imports in certain environments (i think webpack4), so we ended up using internal imports from the `cjs/` folder. The idea is that if you only import the files you are using, instead of the barrel, then you don't need to rely on bundlers for treeshaking. (Also, this only needed to be done in `@itwin/itwinui-react` which gets delivered to the user. Anything else like storybook, tests, etc are completely irrelevant to this conversation and can use module imports with no problems.)

But using these internal cjs imports breaks ESM environments. So in #917, we removed the dependency completely and are now just inlining the icons that we actually need, instead of bringing in the whole package.

Similarly, for ESM compatibility, we don't want to rely on internal imports for illustrations. But `ErrorPage` actually imports all (or most) of the svgs in `@itwin/itwinui-illustrations-react` so it would _not_ get treeshaked anyway. This is why, in #924, we switched to module imports, deprecated the `ErrorPage` component, and introduced a new `NonIdealState` component which does not import all of these illustrations. The idea is that if the user doesn't import `ErrorPage`, then the whole component will be treeshaked. And then if they bring in `NonIdealState`, they can import just the one illustration instead of all.

Both of these changes, combined with https://github.com/iTwin/iTwinUI-icons/pull/49, result in lower bundle sizes for everyone and better ESM compatibility.

As for what this means for the styleguide:
1. There is no need to ever import from `cjs/` anywhere in the repo. Those points should be removed.
2. Illustrations are fine to not mention. Eventually we can remove `ErrorPage` and therefore remove the dependency.
3. We could mention the new way of importing icons, which involves:
  - Manually add the svg component to `utils/icons` and `utils/index.ts`.
  - Import it from `utils`

I think point 3 might not belong in styleguide though.